src/osd.c

```diff
@@ -409,9 +409,7 @@ display_osd(struct output *output)
 	struct wl_array views;
 	wl_array_init(&views);
 	view_array_append(server, &views,
-		LAB_VIEW_CRITERIA_CURRENT_WORKSPACE
```

Comment:
It'd be better to take a step at a time - and only optionally remove this criteria.